### PR TITLE
Search changes

### PR DESCRIPTION
**Pull request by slkendell-mt**

### CODE REVIEW

File: backoffice/src/Helpers/SearchHelpers/DroitQueryBuilder.cs [hunk 1]
```diff
@@ -298,7 +295,7 @@ public static IQueryable<Droit> BuildQuery(DroitSearchForm form, IQueryable<Droi
                     !string.IsNullOrEmpty(d.District) &&
                     (d.District.ToLower().Contains(form.District.ToLower()) || 
                     (usePsql? EF.Functions.FuzzyStringMatchLevenshtein(form.District.ToLower(), d.District.ToLower()) :
-                        SearchHelper.GetLevenshteinDistance(form.District.ToLower(), d.District.ToLower())) < MaxLevenshteinDistance
+                        SearchHelper.GetLevenshteinDistance(form.District.ToLower(), d.District.ToLower())) < 1
```
Comment (slkendell-mt) [inline backoffice/src/Helpers/SearchHelpers/DroitQueryBuilder.cs:298]: Probably worth checking this, might need to be <=

Comment (SamBrownMT) [reply backoffice/src/Helpers/SearchHelpers/DroitQueryBuilder.cs:298]: probably as the distance is how many 'changes' there are if the two strings are laid atop each other, so atm these only returns exact matches I believe.

File: backoffice/src/Models/FormModels/SearchFormModels/DroitSearchForm.cs
Comment (SamBrownMT) [inline backoffice/src/Models/FormModels/SearchFormModels/DroitSearchForm.cs:1]: possibly want to change the variables themselves to Min/Max Depth

Comment (slkendell-mt) [reply backoffice/src/Models/FormModels/SearchFormModels/DroitSearchForm.cs:1]: I'm not sure it's needed, as although it adds to the context for users, it loses the consistency and clarity of From To in the complex query

File: backoffice/src/Helpers/SearchHelpers/DroitQueryBuilder.cs (hunk 1 above)
Comment (slkendell-mt) [reply backoffice/src/Helpers/SearchHelpers/DroitQueryBuilder.cs:298]: Maybe introduce a constant for LowLevenshteinDistance being 2?

Comment (SamBrownMT) [reply backoffice/src/Helpers/SearchHelpers/DroitQueryBuilder.cs:298]: Could be a good idea, especially for maintaining it down the line

**Review by SamBrownMT — APPROVED** [review]

**Pull request merged**